packages/components/src/date-time/README.md

```diff
@@ -75,3 +75,12 @@ A callback function which receives a Date object representing a day as an argume
 
 - Type: `Function`
 - Required: No
+
+### isDayHighlighted
+
+A callback function which receives a moment object representing a day as an argument, and should return a Boolean to signify if the day is visually highlighted in the calendar.
```

Comment:
i don't think we should rely on "moment" as a public API, I believe  you originally proposed a  "events" or  "highlightedDays" prop, why did you  abandon  that  proposal?

Reply:
I like how `events` are more declarative but I wasn't able to figure out how to allow the parent component to know what data it is supposed to put in `events` - eg: which month the user is looking at. 

The underlying library provides callbacks for the "back" and "forward" buttons for listing months but you can also navigate the calendar some other ways which do not have callbacks. Like changing the date manually using the `<select>` elements above. It updates the calendar but we don't have any callback for it.

I like the API now that you are asked for each day, since you can very easily implement a fetching mechanism based on it. It also follows already established pattern with `isInvalidDate`.

What's your take on this?

Reply:
Regarding `moment`, I find it super useful but I understand if we don't want to rely on it. That change should be simple.

Reply:
In fact, we discussed removing moment several times because of its huge bundle size, we just didn't champion it yet.

Reply:
Even the calendar picker library relies fully on it, I think.